src/bump/semver.ts

```diff
@@ -0,0 +1,468 @@
+/**
```

Comment:
This file contains the SemVer bump functions, originally part of `bump.ts` - no changes to code have been applied